danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/MainDangerRunner.kt

```diff
@@ -0,0 +1,138 @@
+package systems.danger.kotlin
+
+import systems.danger.kotlin.json.JsonParser
+import systems.danger.kotlin.models.danger.DSL
+import systems.danger.kotlin.models.danger.DangerDSL
+import systems.danger.kotlin.models.danger.DangerResults
+import systems.danger.kotlin.models.git.FilePath
+import systems.danger.kotlin.sdk.DangerContext
+import systems.danger.kotlin.sdk.Violation
+import java.io.File
+
+internal class MainDangerRunner(jsonInputFilePath: FilePath, jsonOutputPath: FilePath) : DangerContext {
```

Comment:
Danger runner now was split in 3 files: `MainDangerRunner` that contains the private implementation, `MainPlugins` that contains the helpers for registering and using Danger plugins and `MainScript` that contains all the API functions like `message` `warn` and initialisation `val danger=Danger(args)` or `danger(args) {...}` block.

Extensions `onGit` `onGitHub` .. moved to KtxDangerDSL.kt